pkg/win32/GNUmakefile

```diff
@@ -56,6 +56,8 @@ staging-%:
 
 	$(TOPLEVEL)/man/simplecpp -D_WIN32 -DPRECOMPILED  \
 	             -D$(shell echo $* | tr a-z A-Z)      \
+	             -DLONG_GAME_NAME="$(PACKAGE_NAME)"   \
```

Comment:
This rule is used to build staging directories for every game, so I think this means you'll be saying "Chocolate Doom", "chocolate-doom", in cases where you mean "Chocolate Heretic", "chocolate-hexen", etc.?

Reply:
Yes; I'd Mia interpreted what this was used for; I thought it was a top level README in a package of all engines. 

I'm lost in a twisty maze of passages, all alike!